jsk_2016_01_baxter_apc/node_scripts/set_tote_contents_param.py

```diff
@@ -17,5 +17,5 @@
     json_file = args.json_file
 
     data = json.load(open(json_file))
-    rospy.set_param('~tote_contents', data['tote_contents'])
+    rospy.set_param('/tote_contents', data['tote_contents'])
```

Comment:
like this https://github.com/start-jsk/jsk_apc/blob/master/jsk_2016_01_baxter_apc/launch/main_stow.launch#L88